packages/http-server-csharp/src/cli/cli.ts

```diff
@@ -27,10 +29,27 @@ async function main() {
             type: "boolean",
             default: false,
           })
-          .positional("project-directory", {
+          .option("project-name", {
+            description: "The name of the generated project.",
+            type: "string",
+            default: "ServiceProject",
+          })
+          .option("http-port", {
+            description: "The http port for the generated project to use locally",
+            type: "number",
+          })
+          .option("https-port", {
+            description: "The https port for the generated service to listen on locally.",
+            type: "number",
+          })
+          .option("overwrite", {
+            description: "Overwrite existing mock implementations",
+            type: "boolean",
+            default: true,
+          })
+          .option("output", {
             description: "Path to the directory where the project will be created.",
             type: "string",
-            demandOption: true,
           })
           .positional("path-to-spec", {
             description: "The path to the spec to generate a project for",
```

Comment:
nit. "The path to the TypeSpec spec project"

Reply:
It's actually the path to the typespec spec